src/test/java/CommitTest.java

```diff
@@ -0,0 +1,10 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+public class CommitTest {
+    @Test
+    void create_test() {
+        assertThat("String3").isEqualTo("String3");
```

Comment:
```suggestion
    void create_test() {
        assertThat("String4").isEqualTo("String4");
```
이렇게 하면 가독성이 더 좋아집니다.